src/styles/card.css

```
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  max-width: var(--card-max-width);
+  background-color: var( --card-background-color);
+  border: var(--card-border) solid var(--border-color);
+  border-radius: var(--card-border-radius);
+}
+
+.card__header {
+  width: 100%;
+  margin: var(--card-margin);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 19px;
+  padding-bottom: 0;
+  padding-top: var(--card-header-top-gap);
+}
+
+.card__image {
+  height: var(--card-image-height);
+  margin: var(--card-margin);
+  padding: var(--card-padding);
+}
+
+.card__title-section {
+  display: block;
+  width: 100%;
+}
+
+.card__title {
+  margin: var(--card-margin);
+  vertical-align: text-top;
+  padding: var(--min-section-gap);
+  padding-bottom: 0;
+  padding-top: var(--card-padding-top);
+  font-size: var(--card-font-size);
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  font-style: medium;
+  line-height: 18px;
+  color: var(--cardfont-color);
+}
+
+.card__description {
+  margin: var(--card-margin);
+  padding: var(--card-padding);
+  padding-top: 4px;
+  padding-left: var(--min-section-gap);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--card-description-grey);
+  font-family: Roboto, sans-serif;
```

You can just set the font-family for the whole page or at least card, and inherit where needed
```suggestion
  font-family: inherit;
```